packages/salesforcedx-vscode-apex-debugger/src/index.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as vscode from 'vscode';
+
+const initialDebugConfigurations = {
+  version: '0.2.0',
```

Comment:
Do you know if there are any upcoming version of the configurations? I haven't follow the debug configuration space much so I don't know if there is anything new coming up. Just wanted to know if we need to be prepared for any upcoming changes.